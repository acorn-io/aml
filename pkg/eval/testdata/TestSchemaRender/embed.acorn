{
    x: 1
    {
        a: number
        b: 2
    }
    y: {
        {
            a: [1, 2]
        }
    }
}
