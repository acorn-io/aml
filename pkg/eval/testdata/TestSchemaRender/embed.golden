{
  "fields": [
    {
      "name": "x",
      "type": {
        "kind": "number",
        "default": 1
      }
    },
    {
      "name": "a",
      "type": {
        "kind": "number"
      }
    },
    {
      "name": "b",
      "type": {
        "kind": "number",
        "default": 2
      }
    },
    {
      "name": "y",
      "type": {
        "kind": "object",
        "object": {
          "path": "embedded.0.y",
          "fields": [
            {
              "name": "a",
              "type": {
                "kind": "array",
                "array": {
                  "item": {
                    "kind": "number",
                    "default": 1,
                    "alternate": {
                      "kind": "number",
                      "default": 2
                    }
                  }
                }
              }
            }
          ]
        }
      }
    }
  ]
}